catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/SubmitStandingQueryRequestImpl.java

```diff
@@ -411,17 +411,20 @@ public void run() {
                 }
 
                 if (startDate != null) {
-                    if (startDate.getTime() > lastExecutionTime) {
-                        shouldRun = false;
+                    long now = System.currentTimeMillis();
+                    if (startDate.getTime() > now) {
+                        long waitToStart = startDate.getTime() - now;
```

Comment:
I have updated this to wait a configurable maximum amount. Once that time is reached the subscription will start independent of what was passed.